### PR TITLE
Improper positioning of Event Statistics in the mobile view 

### LINKED ISSUE #1286: Improper positioning of Event Statistics in the mobile view
**Bug Description:**
Encountering inconsistent modal behavior and improper positioning UI in the mobile view for Event Statistics.

**Steps to Reproduce:**
1. Navigate to the organization.
2. Click on "Events."
3. Access the Event dashboard.
4. Click on "View Event Statistics."

**Expected Behavior:**
Anticipating a clear and concise presentation of the UI in line with expectations.

**Actual Behavior:**
Observing discrepancies in the UI, specifically with modal positioning, not aligning with anticipated behavior.

**Screenshots:**
Screenshot

**Additional details**

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by duplixx**
**What kind of change does this PR introduce?**
Bug Fixes

**Issue Number:** 

Fixes #1286 

**Did you add tests for your changes?**

**Snapshots/Videos:**

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

### CODE REVIEW

**Comment by Cioppolo14** [comment]
@duplixx Please fix the first comment so that each issue listed automatically closes. The PR_GUIDELINES.md file has details.

**Comment by duplixx** [comment]
@Cioppolo14 i have fixed, Can you check it.
Thanks!!

File: src/components/EventStats/EventStats.tsx [hunk 1]
```diff
@@ -3,8 +3,9 @@ import { Modal } from 'react-bootstrap';
 import { FeedbackStats } from './Statistics/Feedback';
 import { ReviewStats } from './Statistics/Review';
 import { AverageRating } from './Statistics/AverageRating';
-import Stack from '@mui/material/Stack';
+// import Stack from '@mui/material/Stack';
 import styles from './Loader.module.css';
+import styling from './EventStats.module.css';
```
Comment (beingnoble03) [inline src/components/EventStats/EventStats.tsx:8]:
```suggestion
import eventStatsStyles from './EventStats.module.css';
```
Always use this type of naming while importing css.

**Review by beingnoble03 — CHANGES_REQUESTED** [review]
Please address the above comments. Also, upload the after and before video screenshots effects of this PR, in the PR descriptions.

**Comment by duplixx** [comment]
yeah sure @beingnoble03 i'll make changes thanks for review!!

**Review by beingnoble03 — APPROVED** [review]
LGTM!

**Review by noman2002 — APPROVED** [review]

**Pull request merged**